src/PAModelTests/UtilityTests.cs

```diff
@@ -36,16 +36,15 @@ public void TestNotEscaped()
         }
 
         [DataTestMethod]
-        // [DataRow("C:\\Foo\\Bar\\Baz", "C:\\Foo", "Bar\\Baz\\")]
-        // [DataRow("C:\\Foo\\Bar\\Baz", "C:\\Foo\\", "Bar\\Baz\\")]
-        // [DataRow("C:\\Foo\\Bar\\Baz\\", "C:\\Foo\\", "Bar\\Baz\\")]
```

Comment:
This test case should be removed, since the 1st arg is a file and so can never end in \.